core/templates/dev/head/filters/string-utility-filters/camel-case-to-hyphens.pipe.spec.ts

```diff
@@ -0,0 +1,39 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Tests for CamelCaseToHyphens filter for Oppia.
+ */
+
+import { CamelCaseToHyphensPipe } from
+  'filters/string-utility-filters/camel-case-to-hyphens.pipe';
+
+describe('Testing filters', () => {
+  let filterName: CamelCaseToHyphensPipe = null;
```

Comment:
Still confused with the name, infuture are we expecting to use a filter keyword or pipe? (or both)

Reply:
Filter = pipe in angular, we can use any one. Though, the implementation of the filter/pipe would be pipe based.